examples/opentelemetry/otel-collector-config.yaml

```diff
@@ -9,7 +9,9 @@ receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:4317
```

Comment:
are these necessary?

Reply:
Yes without these endpoints I was not able to see the traces even after following the documentation